mlflow/server/js/src/utils/ArtifactUtils.js

```diff
@@ -2,17 +2,21 @@ export class ArtifactNode {
   constructor(isRoot, fileInfo, children) {
     this.isRoot = isRoot;
     this.isLoaded = false;
+    // fileInfo should not be defined for the root node.
```

Comment:
Does this comment pertain to something?

Reply:
Yup, it just means that we don't expect fileInfo to be set on the root `ArtifactNode`.